docker/install_deps.sh

```diff
@@ -47,13 +47,22 @@ if [[ "${TARGETARCH}" == "arm" ]]; then
 fi
 
 # ffmpeg -> arm64
+if command -v ffmpeg >/dev/null 2>&1; then
+    HAS_FFMPEG=1
+else
+    HAS_FFMPEG=0
+fi
+
+if [[ "${HAS_FFMPEG}" == 0 ]]; then
 if [[ "${TARGETARCH}" == "arm64" ]]; then
     # add raspberry pi repo
     gpg --no-default-keyring --keyring /usr/share/keyrings/raspbian.gpg --keyserver keyserver.ubuntu.com --recv-keys 82B129927FA3303E
     echo "deb [signed-by=/usr/share/keyrings/raspbian.gpg] https://archive.raspberrypi.org/debian/ bullseye main" | tee /etc/apt/sources.list.d/raspi.list
     apt-get -qq update
     apt-get -qq install --no-install-recommends --no-install-suggests -y ffmpeg
 fi
```

Comment:
I think this may need to be indented since it is nested if ?

Reply:
yes, agree, needs consistency
But keeping this as is while Work in Progress

Reply:
I'll go ahead and mark the PR as draft as well 👍

Reply:
sounds good, thanks  for the assistance.

I'm finalizing mpp + ffmpeg work, hopefully to get it sorted today
next week to work on adding armnn libs and binaries and then to finish what @gilankpam created in his fork
After all of that - to add predefined preset for ffmpeg and test everything on real hardware

BTW, do we have any solution for how to run arm64 Docker on OSX ( x86_64 )? @NickM-27

Reply:
> BTW, do we have any solution for how to run arm64 Docker on OSX ( x86_64 )? @NickM-27

I am not sure unfortunately. I dev on an M1 Pro based macbook so not sure. You'd probably need to run parallels or something similar